src/core/tsi/ssl_transport_security.cc

```diff
@@ -1541,23 +1541,20 @@ static tsi_result ssl_handshaker_next(tsi_handshaker* self,
   tsi_result status = TSI_OK;
   size_t bytes_written = 0;
   if (received_bytes_size > 0) {
-    unsigned char* remaining_bytes_to_write_to_network_io =
+    unsigned char* remaining_bytes_to_write_to_openssl =
```

Comment:
Do we actually need these two variables in the first place?  Can't we just modify the `received_bytes` and `received_bytes_size` variables as we consume the data?

Reply:
`received_bytes_size` is needed later on when calculating the unused bytes, so we would at least need to copy the initial value of this argument. Rather than doing that, I'm leaning towards keeping the current version since I think it's more readable. More than happy to change it if you disagree though.

Reply:
Okay, this is fine, then.